docs/source/swift-scripting.md

```diff
@@ -35,230 +37,123 @@ The rest of this guide will help you set up a Swift Package Manager executable t
-      | - MyProject // Contains app target source files
-      | - MyLibraryTarget // Contains lib target source files
-      | - MyProjectTests // Contains test files
-    ```
-2. Create a new directory for the Codegen executable by running `mkdir Codegen` in Terminal. Your directory structure should now look like this: 
-
-    ```txt:title=Sample%20Project%20Structure
-    MyProject // Source root
-      | MyProject.xcodeproj
-      | - MyProject // Contains app target source files
-      | - MyLibraryTarget // Contains lib target source files
-      | - MyProjectTests // Contains test files
-      | - Codegen // Contains your Swift Scripting files
-    ```
-3. Using Terminal, change directories into the codegen folder, and initialize an SPM executable by using the following commands:
+You can download the current version of the template from this repo: [https://github.com/apollographql/iOSCodegenTemplate](https://github.com/apollographql/iOSCodegenTemplate)
 
-    ```txt:title=Terminal
-    cd Codegen
-    swift package init --type executable 
-    ```
-    
-    When this command finishes, you'll see that the Codegen folder now has new contents: 
-    
-    ```txt:title=Sample%20Project%20Structure
-    MyProject // Source root
-      | MyProject.xcodeproj
-      | - MyProject // Contains app target source files
-      | - MyLibraryTarget // Contains lib target source files
-      | - MyProjectTests // Contains test files
-      | - Codegen // Contains your Swift Scripting files
-           | Package.swift
-           | README.md
-           | - Sources
-               | - Codegen
-           | - Tests
-               | - CodegenTests
-    ```
-4. Double-click `Package.swift` in this new folder (or run `open Package.swift` in Terminal). This opens the package you've just created in Xcode. 
+When you unzip the downloaded repo, you'll see that there's a folder called **`ApolloCodgen`**. 
 
-5. Update the `dependencies` section to grab the Apollo iOS library:
+If you're using the default target structure for an Xcode project, your project's file structure will look essentially like this in Finder: 
 
-    ```swift:title=Package.swift
-    .package(name: "Apollo",
-             url: "https://github.com/apollographql/apollo-ios.git", 
-             .upToNextMinor(from: "0.29.0"))
-    ```
-  **NOTE**: The version should be identical to the version you're using in your main project.
+```txt:title=Sample%20Project%20Structure
+MyProject // Source root
+  | MyProject.xcodeproj
+  | - MyProject // Contains app target source files
+  | - MyLibraryTarget // Contains lib target source files
+  | - MyProjectTests // Contains test files
+```
 
-    **ALSO NOTE**: Having to specify the name is a workaround for [SR-12110](https://bugs.swift.org/browse/SR-12210). Hopefully once that's fixed, SPM should pick up the name automatically. 
+Drag the `ApolloCodegen` folder in at the same level as your other targets (in Finder, not in Xcode): 
```

Comment:
Not that i've found that makes any sense with how we're trying to make this flexible to use.

The problem is that you can only distribute a library as a package, you can't distribute an executable. This is why `ApolloCodegenLib` is a library, but this executable is in its own project. 

We also want to make it possible for people to use with whatever their setup is, and not have to worry about conforming their project to our assumptions about where things should be. To do that, we'd either have to make parameters for everything supported by the TS codegen (which to me, defeats the point of the whole wrapper since it puts you back into "let's figure out where things are in the filesystem using bash and environment variables!" mode, which is my least favorite mode), or we'd only be able to support a really limited subset of options. 

The other problem is if we shipped this as a compiled binary tied to a version of the library, there's not a great way to control where that gets checked out since that checkout is handled by SPM. We'd be right back where we are with the bash script in terms of having to go folder-diving through derived data to figure out where this is checked out, and that's something I'm explicitly trying to avoid here.